package.json

```diff
@@ -51,8 +50,11 @@
 		"ora": "5.4.1",
 		"prettier": "^2.5.1",
 		"semver": "^7.3.5",
-		"source-map-support": "^0.5.19",
-		"terser": "^5.9.0",
-		"typescript": "^4.5.3"
+		"source-map-support": "^0.5.21",
+		"terser": "^5.10.0",
+		"typescript": "^4.5.5"
+	},
+	"dependencies": {
+		"eslint-plugin-eqeqeq-fix": "^1.0.3"
```

Comment:
Dev dependency?

Reply:
like I said we're removing that, rhys added that not me